charms/mariadb/config.yaml

```diff
@@ -8,6 +8,10 @@ options:
     type: string
     default: defaultdb
     description: The database to create
+  secondary-database:
+    type: string
+    default: katib
+    description: Workaround for hardcoded database name in katib-manager
```

Comment:
Instead of doing this config, it would be much cleaner to add support to the charm for the [`mysql-shared`](https://github.com/openstack/charm-interface-mysql-shared) interface, which would allow the charm to specify the name of the database to create, or even create multiple databases.  You could just drop support for the `mysql` interface altogether and remove the `database` config option as well.  We do still need to get away from sending the same (root) username & password to every related application, as well.

Reply:
Switched to osm-charmers mariadb charm